src/modules/Resize/info.json

```diff
@@ -1,11 +1,22 @@
+
 {
   "name": "resize",
   "description": "Resize image by given percentage value",
   "inputs": {
-    "resize": {
-      "type": "string",
+    "height": {
+      "type": "integer",
+      "desc": "Percentage value of the resize height",
+      "default": "125"
+    },
+    "width": {
+      "type": "integer",
+      "desc": "Percentage value of the resize width",
+      "default": "125"
+    },
+    "checkbox": {
+      "type": "preserve aspect ratio",
       "desc": "Percentage value of the resize",
-      "default": "125%"
+      "default": "0"
```

Comment:
default should still be 125% for making the tests pass also make the default value of checkbox to be one

Reply:
I guess you have seen my initial commit this was a mistake description should be the value of checkbox  and I can't make it's   default 125 as it can be 0 or 1 only.